### DISCUSSION TITLE
Any anticipated release date for GIS/geospatial support?

### DISCUSSION THREAD

**Discussion by geeklisted** [geldata/gel#4307] [General]
One of our main requirements - and the primary reason why we have not yet moved to EdgeDB - is the lack of built-in geospatial support (i.e. within boundary, near, intersecting, etc.) for queries.

On the Roadmap, GIS Extensions are listed for a Post 1.0 release. Since I have not seen any updates or any issues pertaining to geospatial support, I wanted to see where things currently stand in regard to ongoing development.

Thanks.

**Comment by spigelli** [comment]
Also timescale?

**Reply by davidhmays** [reply]
Timescale DB?

**Comment by 1st1** [comment]
We are not working on it right now. It's on our ToDo but in order to expose PostGIS properly we'll have to implement a system for DB extensions in EdgeDB. Most likely we'll focus on that for 4.0.

**Reply by MattHeslington** [reply]
Hello Yuri,

I've just been watching the keynote (for the second time). Is there any news on this?

**Comment by brielov** [comment]
Any news on this?

**Reply by MattHeslington** [reply]
As I understand it, EdgeDB is built on PostgreSQL, which, through the postgris extension among others, has excellent support for geospatial queries, so I really hope it's something which can be incorporated. It would be a 'game-changer' for me

**Comment by elvenking** [comment]
Is there any progress regarding PostGIS support or extension support in general ? @1st1
